app/src/main/java/com/android/streetworkapp/ui/progress/Progression.kt

```diff
@@ -170,6 +170,14 @@ fun ProgressScreen(
               }
               DashboardStateProgression.Training -> {
 
+                if (currentWorkout?.workoutSessions?.isEmpty() == true) {
```

Comment:
You can put : `currentWorkout?.workoutSessions.isNullOrEmpty()` instead of a == true check,
Not really important, resolve if you think its fine like this